content/en/docs/notation/developer/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Developer documentation
```

Comment:
Where there are two duplicated Developer documentation? 
![image](https://github.com/notaryproject/notaryproject.dev/assets/40452856/423be3c5-1f50-4d45-8e15-5d9b1263cb41)

Reply:
@FeynmanZhou  i asked for 2 different developer dirs:
- Developer docs for notation specifically (e.g. plugins)
- Developer docs for Notary (e.g. implementing the spec)

For the secure deploy and experimental features docs, I'm fine with moving them to notation/concepts too.

Reply:
I think both `Developer docs for notation` and
`Developer docs for Notary`  can be subfolders inside a single developer documentation directory. For example:

Developer documentation
-  Developer docs for notation (the secure plugin development, deployment, and experimental features docs would be here)
-  Developer docs for Notary (e.g. implementing the spec)

Please let me know what you think of this. @zr-msft & @FeynmanZhou

Reply:
Considering that the intended audience for `secure-deployment.md` and `experimental-features.md` are users, I could submit another PR to move the docs to `content.en/docs/notation/concepts` as @zr-msft said. 

To avoid duplicating developer documentation, the developer docs for notation and Notary can be grouped under a single directory. Here's an example of how it could be organized:

```
Troubleshooting
Developer documentation
- Notation Developer documentation
  - securely-develop-plugins.md
- Notary Developer documentation
  - implementing-specs.md
  ```      
This way, developers can access documentation for both Notation and Notary within a single directory.

Alternatively, we could have a single directory that includes documentation for both Notation and Notary like so:

```
Troubleshooting
Developer documentation
- securely-develop-plugins.md
- implementing-specs.md
```

please let me know what you think. @FeynmanZhou

Reply:
@Roseline-Bassey I think its best to keep the original idea of:
- `content/en/docs/notation/developer` 
- `content/en/docs/developer`

This allows us to version the docs for the notary project and notation independently

@Roseline-Bassey also, if you are making changes based on feedback in this PR, please add another commit to this PR or amend the current commit in this PR
eg:
> Considering that the intended audience for secure-deployment.md and experimental-features.md are users, I could submit another PR to move the docs to content.en/docs/notation/concepts ...

Reply:
I have submitted a PR to move `secure-deployment.md` and `experimental-features.md` to `content/en/docs/notation/concepts`